test/tags/analyzer/valid.tag

```diff
@@ -1,5 +1,3 @@
-var riot = require('riot')
```

Comment:
why have you removed this line from our tests?

Reply:
This line seems to have been erroneously copied from test/tags/analyzer/invalid.tag as it's the only other tag that also has this line.

Putting require() in the tag files also results in multiple require statements in the output when cli.make() is run against a folder instead of a specific tag, which I don't think should be considered valid. Using the --modular option is the correct way to require() riot.

However, this is unrelated to the purpose of the PR and should be a different request.  I'll remove it from this PR.